File: evaluation/src/main/java/org/arnhold/evaluation/openaire/service/OpenAireController.java
package org.arnhold.evaluation.openaire.service;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/openaire")
public class OpenAireController {

    // https://pidforum.org/t/pids-in-openaire/1747/5

    @GetMapping("test")
    public void test() {
        System.out.println("");
    }

}

File: case-study/src/main/kotlin/org/arnhold/casestudy/context/openaire/OpenAireController.kt
package org.arnhold.casestudy.context.openaire

import org.apache.jena.rdf.model.ModelFactory
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/openaire")
class OpenAireController {
    @Autowired
    var openAireService: OpenAireService? = null

    // https://pidforum.org/t/pids-in-openaire/1747/5
    @GetMapping("test")
    fun test(): String? {
        return openAireService!!.test(ModelFactory.createDefaultModel())
    }
}
